ossfuzz/fuzz_data_producer.c

```diff
@@ -32,24 +32,36 @@ uint32_t FUZZ_dataProducer_uint32(FUZZ_dataProducer_t *producer, uint32_t min,
     rolling >>= 8;
   }
 
-  if (range == 0xffffffff) {
-    return result;
-  }
+  return result;
+}
 
-  return min + result % (range + 1);
+uint32_t FUZZ_dataProducer_uint32(uint32_t seed, uint32_t min, uint32_t max)
```

Comment:
In contrast, this one could be called (for example) `FUZZ_getRange_from_uint32()` or something equivalent.
Note that it doesn't act on any `*producer`.